StripeUICore/StripeUICore/Source/Elements/Section/SectionContainerView.swift

```diff
@@ -32,21 +32,27 @@ class SectionContainerView: UIView {
     }()
 
     lazy var stackView: StackViewWithSeparator = {
-        let view = buildStackView(views: views, theme: theme)
+        let view = buildStackView(views: views, allowLiquidGlassCornerRadius: allowLiquidGlassCornerRadius, theme: theme)
         return view
     }()
 
     /// The list of views to display in a vertical stack
     private(set) var views: [UIView]
+    private let allowLiquidGlassCornerRadius: Bool
```

Comment:
A way to allow components to opt-out of the automatically applied iOS 26 corner radius. Open to different naming!